Site-Reliability-Engineering/L3-Senior-Site-Reliability-Engineer.md

```diff
@@ -0,0 +1,82 @@
+- I identify, qualify and reduce technical debt as appropriate to my team's priorities.
+
+<details>
+<summary>Examples</summary>
+
+- I set up or maintained an automated integration and delivery pipeline.
+- I determined the technical direction within a brown-field project.
+- I built a reusable template for metric dashboards and alerting
+- I made pragmatic decisions in order to ship.
+- People asked me for my opinion when making technical decisions because I had a proven track record of making wise choices.
+
+</details>
+
+## 🌱 Teaching and Mentoring
+
+- I consistently help new hires and more junior engineers to "level up" and become more proficient over time
+- I am fuelling my team's desire to learn.
+- I help others to understand and manage the tradeoffs between aspects of system reliability.
+
+<details>
+<summary>Examples</summary>
+
+- I ran a knowledge sharing session.
+- I mentored a more junior developer and they went on to achieve something they couldn't have before.
+- I trained a team member in how to run incident reviews
+- I shared useful videos/blogs/papers that led to some action.
+- I used reviews as an opportunity to teach and showed others alternate, cleaner ways to implement functionality in an ego-less manner.
+- I helped a team member decide to implement observability tooling before shipping new features so that they could then ship and understand their customers’ production experience
+
+</details>
+
+## 🧭 Culture and Leadership
+
+- I collaborate on technical direction within already-defined projects.
+- I continuously maintain and improve quality across the stack.
+- I am willing to take on the necessary and unglamorous work needed to get things done.
+- I take responsibility for my own mistakes and learn from the mistakes of others.
+- I manage difficult conversations and tactfully challenge others.
+- I improve the productivity and delivery of my team.
+
+<details>
+<summary>Examples</summary>
+
+- I performed regular interviews for engineering candidates, and provided detailed and useful feedback.
+- I took on a significant share of unplanned work and other "housekeeping" tasks.
+- I spotted a contentious issue that could have gone badly and facilitated everyone toward a decision that resolved the situation.
+- I recognised a problem early and got in to fix it even though it wasn't my fault.
+- I wrote a clear and concise proposal that persuaded the team to act on my idea.
+
+</details>
+
+## 🏆 Customer Success
+
+- I ship ideas from conception to production.
+- I advocate for customer needs and motivations.
+- I decompose larger requirements into smaller, more manageable pieces, to deliver functionality in an incremental and iterative manner.
+- I build requirements and technical recommendations based on the real needs of customers.
+
+<details>
+<summary>Examples</summary>
+
+- I led a shaped pitch from idea to production
```

Comment:
I'm not sure `shaped pitch` is the right term any more.

Reply:
I am ok with shaped pitch for the moment. A lot of our confluence info for R&D still mentions it as a tool we can use, even if we don't have formal ShapeUp cycles etc..

Reply:
Yes, let's leave it for now until we decide as a group what the new terms are.